glotaran/utils/test/test_io.py

```diff
@@ -157,3 +158,13 @@ def test_relative_posix_path(tmp_path: Path, rel_file_path: str):
     )
 
     assert rel_result_no_coomon == f"../{rel_file_path}"
+
+
+@pytest.mark.skipif(not sys.platform.startswith("win32"), reason="Only needed for wondows")
```

Comment:
I understand you hold a grudge, but I still suggest calling Windows by it's correct name. ^^

Reply:
Perhaps do a quick investigation on the behavior on platforms other than Windows, e.g. a Mac user that has their data on an external drive or in the cloud.

Reply:
Ups will fix that typo.

As per [documentation](https://docs.python.org/3/library/os.path.html#os.path.relpath), this is a windows only thing.

Reply:
Fixed in 4d6301c0a66e31de369eba0c044c162f2f81c60e